package.json

```diff
@@ -17,12 +17,14 @@
   "scripts": {
     "postinstall": "lerna bootstrap",
     "build": "lerna run compile",
-    "lint": "eslint . --ext .ts --ext .js && lerna run --scope @pipcook/pipcook-board-client lint",
+    "lint": "eslint . --ext .ts --ext .js --fix",
     "pretest": "npm run lint",
     "test": "lerna run --scope @pipcook/pipcook-core test",
     "typedoc": "typedoc"
   },
   "dependencies": {
-    "better-opn": "^1.0.0"
-  }
+    "@types/jasmine": "^3.5.10",
+    "jasmine": "^3.5.0"
+  },
+  "__npminstall_done": false
```

Comment:
What's this for?

Reply:
seems like automatically injected by lerna. I'm gonna remove it

Reply:
Thanks :)